nautobot_ssot/integrations/infoblox/models.py

```diff
@@ -83,6 +88,9 @@ class SSOTInfobloxConfig(PrimaryModel):  # pylint: disable=too-many-ancestors
         verbose_name="Import VLANs",
     )
     infoblox_sync_filters = models.JSONField(default=_get_default_sync_filters, encoder=DjangoJSONEncoder)
+    infoblox_network_view_to_namespace_map = models.JSONField(
+        default=_get_network_view_to_namespace_map, encoder=DjangoJSONEncoder
```

Comment:
similiar to other comment, is this not intended to allow you to overwrite something?

Reply:
Also addressed in #728.